lib/handbook/addon/docs/components/contributor-list/controller.ts

```diff
@@ -0,0 +1,102 @@
+import { action, computed } from '@ember-decorators/object';
+import { A } from '@ember/array';
+import Controller from '@ember/controller';
+import EmberObject from '@ember/object';
+import { faker } from 'ember-cli-mirage';
+
+import Contributor from 'ember-osf-web/models/contributor';
+import { QueryHasManyResult } from 'ember-osf-web/models/osf-model';
+
+type ContribArray = QueryHasManyResult<Contributor>;
+
+export default class DemoContributorList extends Controller {
+    contributors!: ContribArray;
+    node!: EmberObject;
+    nodeWithNonBibliographic!: EmberObject;
+
+    showStep: number = 0;
+
+    init(this: DemoContributorList) {
+        super.init();
+        const node = EmberObject.create({ hasMany: () => ({ value: () => this.makeNContributors(10, true) }) });
```

Comment:
You can generate this data in the mirage default scenario and use the `store` to load it here.